tests/integration/src/test/resources/org/mule/test/integration/locator/component-locator-config.xml

```diff
@@ -14,5 +14,10 @@
         </async>
     </flow>
 
+    <flow name="anotherFlow">
+        <scheduler/>
```

Comment:
I'm not convinced about default frequency, but it makes some sense.  The concern here is the default/implicit scheduler, this is using fixed-freq vs. cron or other scheduler but this is not at all clear.